project-base/storefront/helpers/errors/logException.ts

```diff
@@ -1,10 +1,15 @@
-import { captureException } from '@sentry/nextjs';
+import * as Sentry from '@sentry/nextjs';
```

Comment:
Hmm, why prefer importing whole Sentry package to import each function?

Reply:
Done